### PR TITLE
Adding support for EFM 4.2

### PR DESCRIPTION
**Pull request by vibhorkum**
Adding support for EFM 4.2

### CODE REVIEW

File: roles/setup_efm/tasks/setup_efm.yml [hunk 1]
```diff
@@ -23,6 +23,26 @@
...
+    path: "{{ pg_data }}/pg_wal"
+  register: waldir
+
+- name: Find the source of WAL dir if it's symlink
+  shell: >
+    dirname "{{ waldir.stat.lnk_source }}"
+  when:
+    - waldir is defined
+    - waldir.stat.islnk
+  register: wal_src_location
```
Comment (jt-edb) [inline roles/setup_efm/tasks/setup_efm.yml:37]: I think you could extract parent folder path with the help of the `dirname` jinja filter:
```yaml
- name: Find the source of WAL dir if it's symlink
  set_fact:
     wal_src_location: "{{ waldir.stat.lnk_source | dirname }}"
  when:
    - waldir is defined
    - waldir.stat.islnk
```

**Review by jt-edb — COMMENTED** [review]
Looks good, one minor comment.

File: roles/setup_efm/tasks/setup_efm.yml (hunk 1 above)
Comment (vibhorkumar123) [reply roles/setup_efm/tasks/setup_efm.yml:37]: Interesting. Thank you for sharing this.

Comment (vibhorkumar123) [reply roles/setup_efm/tasks/setup_efm.yml]: Fixed

**Review by jt-edb — APPROVED** [review]

**Pull request merged**